lib/dolly/request.rb

```diff
@@ -40,6 +40,11 @@ def delete resource
       request :delete, full_path(resource), {}
     end
 
+    def attach resource, attachment_name, data, headers = {}
+      data = StringIO.new(data) if data.is_a?(String)
```

Comment:
Looks like CouchRest Base64 encodes inline attachments.
If we are adding attachment functionality to couch we should probably support both inline and standalone.